README.md

```diff
@@ -163,3 +163,4 @@ Please note this email is strictly for reporting security vulnerabilities. For s
 In Q1 2021, Hiro partnered with [Least Authority](https://leastauthority.com/), a leading security consultancy with experience in the crypto space, to audit Hiro Wallet for Web. On April 29th 2021, after addressing the major concerns described in the initial findings, as well as a concluding sign off from the Least Authority team, a final report was delivered.
 
 [Download and read the full report here](https://github.com/blockstack/stacks-wallet-web/blob/main/public/docs/least-authority-security-audit-report.pdf)
+timtest123456
```

Comment:
```suggestion
```

Reply:
This wasn't suppose to be included in the the PR I was testing my github ssh connection in a prior branch. For some reason some commits come up as TW without my avatar, I fixed the issue but the test commit got pulled into this PR. I can fix this as well.

Reply:
> I wonder if there's a way to test this functionality, but without checking the exact error string. If we change the wording it'll fail, which isn't super helpful. Do you have any ideas how we might do this?
> 

The quickest way to check for the assertion is with the string, it gets the job done and gives us some protection from this P1 error coming back in a regression which was my first concern but we can definitely. Improve these test in iterations  with multiple PRs as we add more test.